### PR TITLE
Fixed focus and trap by moving getFocusables call to drawer open

### PR DESCRIPTION
**Pull request by tsrahm**
Mallory, this change fixed focus and trap for Notifications for me locally. Please verify.

### CODE REVIEW

**Comment by StommePoes** [comment]
will check!

File: src/js/Drawer.js [hunk 1]
```diff
@@ -273,7 +274,7 @@ function handleClick(e, target, Drawer) {
       drawerElements = document.querySelectorAll(drawerName);
 
   // Is it actually possible to have multiple drawers per trigger??
-  // Certainly can't be with anchor triggers. 
+  // Certainly can't be with anchor triggers.
```
Comment (wyseguyonline) [inline src/js/Drawer.js]: Can we lose these comment lines?  These seem like they should either be part of a discussion somewhere or changed to explain why something was done (if it isn't obvious from the code).

Comment (StommePoes) [inline src/js/Drawer.js]: This is what you get when a dev (me) is asked to make changes to a component where I have zero idea how it's being used by other teams and am pretty sure my changes are breaking someone, somewhere. Where should I move this? It's not explaining what I did, but indeed showing I'm aware I could be destroying someone's life somewhere :P

**Comment by StommePoes** [comment]
This looks like a good change.

I am not sure why it's failing tests, these tests are testing things that have nothing to do with Drawer and shouldn't be testing them (avatar height, slider stuff...). @umahaea ?

**Comment by umahaea** [comment]
CIs passing, so good with QA.

**Comment by umahaea** [comment]
@StommePoes - the builds were aborted for some reason, may be to clear the queue for something else. I restarted the builds and they pass.

**Comment by umahaea** [comment]
3 approvals, good to merge.

**Pull request merged**